tests/session.test.ts

```diff
@@ -237,13 +237,31 @@ test.describe('Session', () => {
 		await page.getByText('Run').click();
 		await expect(page.locator("pre")).toBeVisible();
 		await expect(page.locator("code")).toBeVisible();
-		await expect(page.getByTitle('Copy')).toHaveCount(5);
+		await expect(page.getByTitle('Copy', {exact: true})).toHaveCount(5);
 
 		await page.locator("pre").hover();
-		await page.getByTitle('Copy').last().click();
+		await page.getByTitle('Copy', {exact: true}).last().click();
 		expect(await page.evaluate(() => navigator.clipboard.readText())).toEqual("def calculate_odds(emma_age, emma_height, emma_weight, emma_experience, jessica_age, jessica_height, jessica_weight, jessica_experience):\n    emma_stats = {'age': emma_age, 'height': emma_height, 'weight': emma_weight, 'experience': emma_experience}\n    jessica_stats = {'age': jessica_age, 'height': jessica_height, 'weight': jessica_weight, 'experience': jessica_experience}\n    \n    # Calculate the differences between their stats\n    age_difference = abs(emma_stats['age'] - jessica_stats['age'])\n    height_difference = abs(emma_stats['height'] - jessica_stats['height'])\n    weight_difference = abs(emma_stats['weight'] - jessica_stats['weight'])\n    \n    # Return the differences as a tuple\n    return (age_difference, height_difference, weight_difference)\n");
 	});
 
+	test('can copy the whole session content to clipboard', async ({ page }) => {
+		await page.goto('/');
+		await page.evaluate(() => navigator.clipboard.writeText(""));
+		await chooseModelFromSettings(page, MOCK_API_TAGS_RESPONSE.models[0].name);
+		await page.getByText('Sessions', { exact: true }).click();
+		await mockCompletionResponse(page, MOCK_SESSION_1_RESPONSE_1);
+		await page.getByTestId('new-session').click();
+		await expect(page.getByTitle('Copy session')).toHaveCount(0);
```

Comment:
After removing the custom `title` attribute you can target the "Copy session" button with:

```typescript
await expect(page.locator(".header").getByTitle('Copy')).toHaveCount(0);
```

Reply:
Also, minor formatting issue, can you add an empty line after this assertion?
I'd like to group interactions + assertions in "blocks" separated by empty lines.